pkg/net/http/server.go

```diff
@@ -89,6 +90,15 @@ func (constructor ServerConstructor) Annotate() fx.Option {
 	)
 }
 
+type apertureLogger struct {
+	logger *log.Logger
+}
+
+func (al *apertureLogger) Write(p []byte) (n int, err error) {
+	al.logger.Debug().Msg(string(p))
+	return len(p), nil
+}
+
```

Comment:
Our existign logger already implements Write interface and can be passed directly.